demo.md

```diff
@@ -0,0 +1,24 @@
+## Demo script:
+
+- ERC-4337 compatible Smart Contract wallet
+- Unique feature: gasless off-chain txs with instant finality enabled by HTLC technology, as used in Bitcoin Lightning
```

Comment:
As mentioned in standup, our contribution is a novel way to hybridize thees two without needing any on chain transactions to move funds in or out of a layer 2 system.